### DISCUSSION TITLE
Search Box stopped working, sharing how I seemed to fix it (SOLVED)

### DISCUSSION THREAD

**Discussion by Gargantubrain** [valinet/ExplorerPatcher#2566] [Q&A]
This morning my laptop booted up and I noticed that typing on the Search menu was no longer working. I use this quite often to just tap the Windows key and start typing like uninstall or whatever.
Not sure where to post this, I don't see another thread that matches the issue. It took me a good half hour of my life so hoping to save anyone else some time. Feel free to merge or move to FAQ or whatever. Thanks all!

What DID work: Logging in as my Administrator account instead of as myself with Standard User role. When I logged in as Administrator, ExplorerPatcher popped up a box indicating that it was downloading Symbols for the build of Windows. After that was all done I rebooted and everything was good on my standard user again.

What did NOT work: any number of settings in ExplorerPatcher or system Settings. i.e. turning on the Search button or box, updating ExplorerPatcher, restarting File Explorer, etc. It was like the search button just died and couldn't be clicked.

**Comment by pyrates999** [comment]
So it was caused by being unable to download symbols: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Reply by Gargantubrain** [reply]
Thanks, that's a good read actually. I did notice that the wiki on Symbols does not mention that when logged in as a Standard level user that the Symbols won't be downloaded -- so my suggestion to login as an actual Administrative user seems to be a valid Public Service Announcement. Again by sharing this I'm hoping to save anyone else going through 30 minutes of troubleshooting and rebooting if they notice the Search Bar (or the Windows-X menu as the Symbols wiki suggests) doesn't seem to be working for them.
